pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
   'deepdiff>=5.6.0',
   'devtools>=0.6',
   'frozendict>=2.3',
-  'gridtools-cpp>=2.3.1,==2.*',
```

Comment:
not strictly needed to bump the min version, but since gridtools-cpp doesn't have external dependencies it's a safe choice.